plaso/output/mediator.py

```diff
@@ -95,25 +96,27 @@ def GetFormattedSources(self, event):
       event (EventObject): event.
 
     Returns:
-      A tuple of the short and long source string. If no event formatter
-      to match the event can be found the function returns a tuple
-      of None, None.
+      tuple(str, str)|tuple(None, None): A tuple of the short and long source
```

Comment:
Per previous discussion we opted to omit to use None in the docstrings as a type but as a value. Undo these changes (repeat elsewhere). Also technically None is not a type, seeing the type of None is NoneType. 

I've added a section to the style guide (as a reminder): https://github.com/log2timeline/plaso/wiki/Style-guide#none-type

If you disagree let's chat.